packages/shared/src/telemetry/clientCache.ts

```diff
@@ -0,0 +1,97 @@
+import type { TelemetryEventRaw } from './types';
+
+type TtlInMilliseconds = number;
+
+const DEFAULT_CACHE_TTL_MS = 1000; // 24 hours
```

Comment:
Is this value just for testing? Or is the comment wrong? 👀

Reply:
Ooops... I was testing it and ended pushing the wrong value.